lib/config/scheme.js

```diff
@@ -36,5 +37,12 @@ module.exports = root(
 
         url: option({
             validate: shouldBeString
+        }),
+
+        renderer: section({
```

Comment:
Oh, the documentation says `rendering`. That's probably better, so I'll change it to that.